pkgs/development/python-modules/skein/skeinjar.nix

```diff
@@ -1,21 +1,19 @@
-{ callPackage, stdenv, maven, src, version }:
+{ fetchurl, unzip, stdenv, version, jarHash }:
 
-let
-  skeinRepo = callPackage ./skeinrepo.nix { inherit src version; };
-in
 stdenv.mkDerivation rec {
   pname = "skein.jar";
+  inherit version;
 
-  inherit version src;
+  src = fetchurl {
+    url = "https://files.pythonhosted.org/packages/py2.py3/s/skein/skein-${version}-py2.py3-none-any.whl";
+    hash = jarHash;
+  };
 
-  nativeBuildInputs = [ maven ];
-
-  buildPhase = ''
-    mvn --offline -f java/pom.xml package -Dmaven.repo.local="${skeinRepo}" -Dskein.version=${version} -Dversion=${version}
-  '';
+  dontUnpack = true;
+  dontBuild = true;
 
   installPhase = ''
-    # Making sure skein.jar exists skips the maven build in setup.py
-    mv java/target/skein-*.jar $out
+    ${unzip}/bin/unzip ${src}
+    mv ./skein/java/skein.jar $out
   '';
```

Comment:
Misses meta.

Reply:
No, this is intentional. [skeinjar.nix](https://github.com/NixOS/nixpkgs/pull/165226/files#diff-6d9fd5cfe0ab3322d01a11c07142311d2199da43f77dac9ea78d95aa1e6f4dd5) is not a top level package, it's just a mkDerivation to fetch a whl from pypi and extract a file out of it. It's used in [default.nix:36](https://github.com/NixOS/nixpkgs/blob/8fc621efab8142d10632e01bcf55018831ecfa88/pkgs/development/python-modules/skein/default.nix#L36).